CONTRIBUTING.md

```diff
@@ -2,29 +2,101 @@
 
 ## Dependencies
 
-We use `poetry` to manage the [dependencies](https://github.com/python-poetry/poetry).
-If you dont have `poetry`, you should install with `make poetry-download`.
+The easiest way to have the environment setup is using conda.
+Conda is the most recommended package manager for scientific
+and data python stack.
 
-To install dependencies and prepare [`pre-commit`](https://pre-commit.com/) hooks you would need to run `install` command:
+If you don't have it installed, you can install it via mambaforge:
+https://github.com/conda-forge/miniforge#mambaforge
+
+Mamba is a fastest layer on top of conda that will fast the way
+conda works.
+
+When you have the conda installed, you can proceed and create
+a new conda environment for epigraphhub.
+
+```bash
+mamba env create --file conda/dev.yaml
+```
+
+This command will create a new conda environment with some development
+dependencies for epigraphhub. In order to proceed, you will need first
+activate your environment:
+
+```bash
+conda activate epigraphhubpy
+```
+
+Now, you can install the epigraphhub dependencies with:
+
+```bash
+poetry install
+```
+
+## Configuration file
+
+`epigraphhub_py` needs a configuration file in order to access the database.
+
+After you have the library installed, you can run the following command
+(just an example):
+
+```bash
+epigraphhub-config \
```

Comment:
the `.env` file is not a good approach for libraries. it is just good for devops.
for example,  superset uses its own configuration python module, we can use .env files .. but it is just for devops setup.

it is common for libraries that uses credentials or configuration file to store that in a file inside a file at `~/.config/`.

I was kind of inspired by: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html

but instead of using a regular configuration file (like .ini) I am using a yaml file.

I can add to the documentation some text about the structure of the file.

Reply:
Agree on these points. 
Configuration in libraries are not very common indeed. 
In our case, I am just worried about making it as simple as possible for the user to install the library and start using it.